### DISCUSSION TITLE
Change in languages.toml rust-analyzer's cargo.target values to target_os="android" target_family="unix"

### DISCUSSION THREAD

**Discussion by boroicamarius** [helix-editor/helix#4638] [Q&A]
I currently do not know how to do this and I actually need the syntax highlighting of rust analyzer for unix families,
![image](https://user-images.githubusercontent.com/64723102/200386088-ad20e1a1-aae8-4ffa-81d7-35b16174fd15.png)
I currently have this written in languages.toml but doesnt actually work.

Please leave a full example for the languages.toml Thanks :D

**Comment by the-mikedavis** [comment]
From the rust-analyzer docs, it looks like there's a `cargo.target` option which takes a target triple string rather than separate `target_family`/`target_os` options: https://rust-analyzer.github.io/manual.html#configuration
